rfcs/text/0000-create-file-elf.md

```diff
@@ -0,0 +1,155 @@
+# 0000: Create the ELF sub-field of the File fieldset
+
+- Stage: **1 (proposal)**
+- Date: **TBD**
+
+Create the Executable Linkable Format (ELF) sub-field, of the `file` top-level fieldset. This document metadata can be used for malware research, as well as coding and other application development efforts.
+
+## Fields
+
+**Stage 0**
+
+This RFC is to create the ELF sub-field within the `file.` fieldset. This will include 25 sub-fields.
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| elf.creation_date | date | Extracted when possible from the file's metadata. Indicates when it was built or compiled. It can also be faked by malware creators. |
+| elf.exports | nested | List of exported element names and types. |
+| elf.exports.name | keyword | Name of exported symbol |
+| elf.exports.type | keyword | Type of exported symbol |
+| elf.segments | keyword | ELF object segment list. |
+| elf.header | nested | Header information of the ELF file. |
```

Comment:
I don't believe this needs to be `nested` because we're not working with multiple headers (nested is just used when you have multiples of something, i.e. segments, etc and need to model complex properties of each subdocument).